web/src/app/app.module.ts

```diff
@@ -34,6 +35,7 @@ import { AppCardFormComponent } from './components/app-card-form/app-card-form.c
     BrowserModule,
     FormsModule,
     HttpClientModule,
+    DragDropModule,
```

Comment:
In Our case, our whole app is drag and drop. but in real time. one specific block will be drag-able and drop-able